src/loaddb/load_server_loader.cpp

```diff
@@ -700,7 +700,7 @@ namespace cubload
 
     insert_errors_filtered = has_errors_filtered_for_insert (m_session.get_args().m_ignored_errors);
 
-    if (insert_errors_filtered)
+    if (insert_errors_filtered || !HA_DISABLED ())
```

Comment:
We should leave a comment for the case "!HA_DISABLED ()".

Reply:
Where is the comment?

Reply:
It's a suggestion. How about leaving a comment for this?

Reply:
Oh. I see.

But I think it is too long story to explain using comments. It would better read the description of jira issue. I will think about the short comment before merging. Thanks.